scripts/taskcluster-android.sh

```diff
@@ -2,56 +2,4 @@
 
 set -euvx
 
-# libtool/automake/autoconf are needed for the patchelf build (The project is
-# small enough (one file) that this is probably not necessary, but it's easy
-# enough to just do it the right way).
-apt-get update -qq && apt-get install zip libtool automake autoconf -y
-
-mkdir -p .cargo
-yes | cp -rf scripts/taskcluster-cargo-config .cargo/config
-pushd libs/ && ./build-all.sh android && ./build-patchelf.sh && popd
-
-declare -A android_targets
-android_targets=(
-  ["x86"]="i686-linux-android"
-  ["arm"]="armv7-linux-androideabi"
-  ["arm64"]="aarch64-linux-android"
-)
-
-if [ "$#" -eq 0 ]
-then
-  selected_targets=(x86 arm arm64)
-else
-  for target_arg in "$@"
-  do
-    [[ -z "${android_targets[$target_arg]+yes}" ]] && echo "Unrecognized target $target_arg. Supported targets: ${!android_targets[@]}" && exit 1
-    selected_targets=("${selected_targets[@]}" $target_arg)
-  done
-fi
-
-echo "Building selected targets: ${selected_targets[@]}."
-
-cd /build/application-services
-
-ORIG_PATH="$PATH"
-for target in "${selected_targets[@]}"
-do
-  PATH="$ANDROID_NDK_TOOLCHAIN_DIR/$target-$ANDROID_NDK_API_VERSION/bin:$ORIG_PATH"
```

Comment:
How does `$ANDROID_NDK_TOOLCHAIN_DIR` work now? Is it still used?

Reply:
It is set by https://github.com/mozilla/application-services/blob/9a2a57d2f1b681a60af8000b2e8758c4de6cfcc8/libs/android_defaults.sh and still used I believe.

Reply:
Correct -- it's magic that `rust-android-gradle` plugin knows about.  One more thing to document!